ivy/functional/frontends/torch/tensor.py

```diff
@@ -281,6 +283,11 @@ def bitwise_and(self, other):
     def bitwise_or(self, other, *, out=None):
         return torch_frontend.bitwise_or(self._ivy_array, other)
 
+
+    def bitwise_or_(self, other, *, out=None):
```

Comment:
I think you've misunderstood what I'm suggesting. I'm saying you should add this `@with_supported_dtypes({"1.11.0 and below": ("integer",)}, "torch")` here.